cssdb.settings.json

```diff
@@ -74,13 +79,9 @@
     "example": "a {\n  break-inside: avoid;\n  break-before: avoid-column;\n  break-after: always;\n}",
     "mdn_path": [
       "css.properties['break-after'].paged_context",
-      "css.properties['break-after'].paged_context.always",
       "css.properties['break-after'].paged_context.page",
-      "css.properties['break-after'].paged_context.recto",
       "css.properties['break-before'].paged_context",
-      "css.properties['break-before'].paged_context.always",
       "css.properties['break-before'].paged_context.page",
-      "css.properties['break-before'].paged_context.recto",
```

Comment:
The plugin does all these things and browsers can support these today.
But the fallbacks don't actually work as far as I can tell.

- `always` is deprecated
- `recto` never shipped, so also not available as a fallback with the other property name.